infra/package.json

```diff
@@ -17,7 +17,7 @@
     "@types/jest": "^27.5.2",
     "@types/node": "^18.16.2",
     "@types/prettier": "^2.7.2",
-    "aws-cdk": "2.77.0",
+    "aws-cdk": ">=2.80.0",
```

Comment:
The identity pool construct version needs to match this, which is why I was pinning. I get totally lost with npm version constraints, but I wasn't able to unpin this without getting version issues with that.

Reply:
The `aws-cdk-lib` dependency should also match this.

Reply:
Thanks for info on both comments, will have to look into this more if this is breaking.